packages/angular-meteor-data/modules/angular-meteor-collection.js

```diff
@@ -34,12 +34,7 @@ angularMeteorCollection.factory('AngularMeteorCollection', [
       // (autorun for cursor.fetch).
       data._hDataAutorun = null;
 
-      if (angular.isDefined(collection)) {
-        data.$$collection = collection;
-      } else {
-        var cursor = curDefFunc();
-        data.$$collection = $meteorUtils.getCollectionByName(cursor.collection.name);
-      }
+      data.$$collection = collection;
```

Comment:
@lekev The thing is that we need this `if` statement so you could define both a cursor or a string.
How do you handle defining a Cursor from a string here?